src/views/Billing/Components/table-services-products.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <DataTable
+    v-model:expandedRows="expandedRows"
+    :value="props.listProduct"
+    rowGroupMode="subheader"
+    groupRowsBy="region"
+    class="w-full sm:w-1/2 h-fit"
+    :pt="{
+      thead: { class: 'hidden' },
+      rowexpansioncell: 'p-0'
+    }"
+  >
+    <template #header>
+      <span class="text-base font-medium">Service and Products Charges</span>
+    </template>
+    <Column
+      expander
+      class="w-0"
+      :pt="{
+        rowToggler: { class: 'border-none' }
+      }"
+    />
+    <Column
+      field="service"
+      header=""
+      class="font-medium !text-base"
+    ></Column>
+    <Column
+      field="value"
+      header=""
+      class="font-medium !text-sm"
+      bodyStyle="text-align:right"
+    ></Column>
+    <template #expansion="slotProps">
+      <div class="p-0 m-0">
+        <DataTable
+          v-for="(expansionColuns, index) in slotProps.data.descriptions"
+          :key="index"
+          :value="expansionColuns.data"
+          class="border-none -mt-0.5 !text-sm text-color-secondary rounded-none"
+        >
+          <Column :pt="{ root: { class: 'pl-12' } }"></Column>
```

Comment:
detalhe (não bloqueante)

Usar o auto fechamento em componentes que não tem conteúdo dentro
```suggestion
          <Column :pt="{ root: { class: 'pl-12' } }" />
```

Reply:
Ajustado